### ISSUE TITLE
Updating version under linux very manual

### ISSUE THREAD

**Issue by R-i-c-h-B** [Microsoft/azuredatastudio#2974]
- Azure Data Studio Version: 1
Linux Fedora.

This request is about maintenance under linux in a corporate environment.  
Currently it seems we need to check against the downloads page to look for updates, download the rpm then install it.
If I have 20 or 30 db team members all doing the same thing, that's a quite large amount of bandwidth and time associated with the updates.  
We have a corporate updates repository, which happily syncs the yum repos from https://packages.microsoft.com/yumrepos/ and enables more automated updates with minimal downloads.

Is there a plan to add azuredatastudio to this, in the same way that vscode is already enabled there?

Cheers

**Comment by kburtram** [comment]
@R-i-c-h-B we're planning to on-board Azure Data Studio onto the official MS package repos for RPM and DEB files.  Mostly we've been waiting for the community to ask (thanks! 😄) and then figure out when to schedule it.  Our next acquisition task is to support per-user setup on Windows, and then improving the Linux acquisition experience would likely come after that.  Hopefully we can have something available for the Dec. or Jan release.

**Comment by lucasyvas** [comment]
Looking forward to this too. The very regular releases make it even trickier to stay on top of it - I'm sure I could write a script to check and update for now!

If this ever gets the vscode style auto changelogs to accompany it that would make a pretty nice experience.

**Comment by spoonerandrew** [comment]
Adding my vote to this as well. Similar process to VSCode would make this a smoother install/upgrade process.

**Comment by tappoz** [comment]
Hoping to see this as well. I am waiting for the Debian/Ubuntu repository. 

Trying to figure out the deb package filename from the URL redirection when automating this installation using something like `wget --content-disposition https://go.microsoft.com/fwlink/?linkid=2038405 > installation_azure_data_studio.txt` is not as handy as I hoped.

Thanks

**Comment by mdx0111** [comment]
Looking forward to see this happening sooner.

Thanks for this great product.

**Comment by oappelt** [comment]
This seems to be about a year overdue.
Any actual plans to get this done? This has been working fine for VSCode for years.

**Comment by kburtram** [comment]
@oappelt we were discussing this a couple days ago.  We plan to have the DEB and RPM published into the MS repos within the next couple releases.

**Comment by mdx0111** [comment]
@kburtram Hi Karl,
I was wondering to ask if you have made any progress on this please.

Many many thanks. :)

**Comment by jaapie** [comment]
Hello. Are there any updates? Updating on Ubuntu is moderatley painful without package repo support. Having said that I love ADS!

**Comment by jaapie** [comment]
While we wait for this to be implemented, I created a script that uses `curl`, and `hxnormailze` and `hxselect` from the `html-xml-utils` library to get the download link from the ADS download page, `wget` to download the `.deb` and `dpkg` to install it. You can find it in a gist [here](https://gist.github.com/jaapie/8e0091577301decf4d79fcf51874a708). Works on Ubuntu right now because that's what I am using.

**Comment by craigs100** [comment]
Similar to this, still not fixed?
https://github.com/microsoft/azuredatastudio/issues/21507

**Comment by jaapie** [comment]
No this issue is relating to the fact that ADS is not able to be updated through a package manager, similar to the way VSCode is. It's much easier to update.

**Comment by craigs100** [comment]
Exactly, you have to manually update it. If it was done how VSCode is done, it would be great. Don't people talk to each other in MS? ;)
